src/lib.rs

```diff
@@ -131,16 +143,50 @@ impl DiskImageBuilder {
         Ok(())
     }
 
+    #[cfg(all(feature = "bios", feature = "embedded_binaries"))]
+    /// Create an MBR disk image for booting on BIOS systems.
+    pub fn create_bios_image(&self, image_path: &Path) -> anyhow::Result<()> {
+        const BIOS_STAGE_3_NAME: &str = "boot-stage-3";
+        const BIOS_STAGE_4_NAME: &str = "boot-stage-4";
+        let stage_3 = FileDataSource::Data(BIOS_STAGE_3.to_vec());
```

Comment:
The `to_vec` call copies the data to the heap whenever this function is invoked. I think we could avoid this copy by introducing a new `FileDataSource::Bytes(&'static [u8])` variant, which could be used with statics and const data.